integration/cli-tests/src/test/java/util/TestUtils.java

```diff
@@ -23,13 +23,26 @@
 import java.io.InputStreamReader;
 import java.util.ArrayList;
 import java.util.List;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.micro.integrator.cli.Constants;
 
 public class TestUtils {
 
+    private static final Log LOG = LogFactory.getLog(TestUtils.class);
+
     private TestUtils() {
 
     }
 
+    public static void login() throws IOException {
+        LOG.info("Logging in to use Micro Integrator Management API");
+        // mi remote login admin admin
+        String[] arguments = new String[]{getMIBuildPath(), Constants.REMOTE, Constants.LOGIN,
+                Constants.DEFAULT_USERNAME, Constants.DEFAULT_PASSWORD};
+        runCommandWithArgs(arguments);
+    }
+
```

Comment:
Shall we try log out as well after class and try to invoke something after logout and check whether it fails

Reply:
Updated.